packages/react-components/react-button/src/components/Button/Button.types.ts

```diff
@@ -13,6 +13,11 @@ export type ButtonSlots = {
   icon?: Slot<'span'>;
 };
 
+/**
+ * A button supports different sizes.
+ */
+export type ButtonSize = 'small' | 'medium' | 'large';
```

Comment:
_nit:_ We call this `AvatarSizes` in `Avatar`, so we should call it `ButtonSizes` to be consistent

```suggestion
export type ButtonSizes = 'small' | 'medium' | 'large';
```

Reply:
IMO its a mistake on the react-avatar package side! I'll open up a PR to deprecate an properly call it `AvatarSize` this is a single entity, it makes no sense to have it in plural just because it's an union of values.